core/domain/user_query_jobs_one_off.py

```diff
@@ -46,7 +46,7 @@ def entity_classes_to_map_over(cls):
         return [user_models.UserSettingsModel]
 
     @staticmethod
-    def map(user_settings_model):
+    def map(user_settings_model):  # pylint: disable=too-many-return-statements
```

Comment:
As far as possible, never introduce disable pragmas. You can instead probably generalize the query_criteria_satisfied logic to cover some of the clauses above it.